src/custom/hooks/useRefetchFee.ts

```diff
@@ -1,15 +1,15 @@
 import { useCallback } from 'react'
-import { useAddFee, useClearFee } from '../state/fee/hooks'
-import { registerOnWindow } from '../utils/misc'
-import { FeeQuoteParams, getFeeQuote } from '../utils/operator'
+import { useUpdateQuote, useClearQuote } from 'state/price/hooks'
+import { registerOnWindow } from 'utils/misc'
+import { FeeQuoteParams, getFeeQuote } from 'utils/operator'
 
 /**
  * @returns callback that fetches a new quote and update the state
  */
-export function useRefetchFeeCallback() {
-  const addFee = useAddFee()
-  const clearFee = useClearFee()
-  registerOnWindow({ addFee })
+export function useRefetchQuoteCallback() {
+  const updateQuote = useUpdateQuote()
+  const clearQuote = useClearQuote()
+  registerOnWindow({ addFee: updateQuote })
```

Comment:
i would change the key name here too but nbd

Reply:
cool doing this in my new PR